packages/twenty-server/src/modules/workflow/workflow-builder/workflow-step/workflow-version-step.workspace-service.ts

```diff
@@ -578,6 +593,37 @@ export class WorkflowVersionStepWorkspaceService {
           },
         };
       }
+      case WorkflowActionType.AI_AGENT: {
+        const newAgent = await this.agentService.createOneAgent(
+          {
+            name: 'AI Agent Workflow Step',
+            description: 'Created automatically for workflow step',
+            prompt: '',
+            modelId: 'gpt-4o',
```

Comment:
What is the point of isDefault? Seems like we can remove it since you're setting a default here?